pydantic/mypy.py

```diff
@@ -799,6 +802,14 @@ def add_initializer(self, fields: list[PydanticModelField], config: ModelConfigD
                 use_alias=use_alias,
                 is_settings=is_settings,
             )
+            if is_root_model:
+                if len(fields) > 1:
+                    pass
+                    # error_extra_fields_on_root_model(self._api)
```

Comment:
What information should I pass as context here to this custom error func?

Reply:
Try `cls.info`